packages/suite/src/components/guide/GuidePanel.tsx

```diff
@@ -12,19 +13,51 @@ import {
     Feedback,
 } from '@guide-views';
 import { useGuide } from '@guide-hooks';
+import * as guideActions from '@suite-actions/guideActions';
+import { GUIDE_ANIMATION_DURATION_MS } from '@suite/hooks/guide/useGuide';
+
+const fullHeightStyle = css`
+    position: absolute;
+    top: 0;
+    right: 0;
+    z-index: ${variables.Z_INDEX.GUIDE_PANEL_BESIDE_MODAL};
+`;
+
+const smoothBlur = keyframes`
+    from {
+        backdrop-filter: blur(0px);
```

Comment:
I think this css attribute is not supported in Firefox by default. Just check that it looks okay there or try to find another solution.

Reply:
Will fix, assigned #2165 to myself too

Reply:
done

Reply:
Sorry. We were investigating this with @marekrjpolak and we think that it does not make sense to fix this `backdrop-filter` issue. Firefox is used only by a small number of users and it unnecessarily complicates our codebase 😱. 

I think we can also close this issue #2165

cc @matejkriz

Reply:
it would be too difficult to maintain
![Screenshot 2022-03-15 at 14 37 09](https://user-images.githubusercontent.com/33235762/158389739-49027873-aa04-40f6-a9f9-e6cd6064e994.png)